internal/provider/provider.go

```diff
@@ -117,14 +122,53 @@ func (p *pingfederateProvider) Schema(_ context.Context, _ provider.SchemaReques
 				MarkdownDescription: "Path for PingFederate Admin API. Default value can be set with the `PINGFEDERATE_PROVIDER_ADMIN_API_PATH` environment variable. If no value is supplied, the value used will be `/pf-admin-api/v1`.",
 				Optional:            true,
 			},
+			"access_token": schema.StringAttribute{
```

Comment:
Why have this separate from the oauth nested attribute? We probably should compare with pingone and/or other providers like Scott mentioned to follow a typical pattern.

Maybe username and password should be in a "basicauth" nested attribute? (though that would be breaking so it might be something that has to wait).

Reply:
we can separate all auths in `v1.0.0`